pykafka/protocol.py

```diff
@@ -1550,6 +1570,12 @@ class OffsetFetchResponse(Response):
             Metadata => string
             ErrorCode => int16
     """
+    API_KEY = 9
+
+    @classmethod
+    def get_versions(cls):
+        return {0: OffsetFetchResponse, 1: OffsetFetchResponse, 2: OffsetFetchResponse}
```

Comment:
`OffsetFetchResponse` v2 adds a response-level `error_code` parameter that should be handled by a new subclass.

Reply:
This can probably be handled similarly to how it's done in the `MetadataResponse` [subclasses](https://github.com/Parsely/pykafka/blob/5b99db27107502f6be3411df672a85eca609d6cf/pykafka/protocol.py#L686)